msr/msre/multiphysics_core_model/steady_state/tests

```diff
@@ -0,0 +1,23 @@
+[Tests]
+  # Syntax checks
```

Comment:
add a test that runs very first using the `working_directory` of ../mesh and generates the mesh

Reply:
Done